src/Shared/Application/EventListener/QueryParameter/Pagination/ExplicitValueEvaluator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Shared\Application\EventListener\QueryParameter\Pagination;
+
+use function is_array;
+use function is_string;
+use function trim;
+
+final class ExplicitValueEvaluator
```

Comment:
Please review all classes and inside EventListener and check if they should be here
Because inside eventlistener directory we should have only event listeners 
You need to move all files that are not event listeners in the different directories